tools/webhook_test.py

```diff
@@ -125,6 +145,46 @@ def int_or_default(input_parm):
         raw_input(), payload["message"][input_parm])
 
 
+def get_gym_info(gym_id):
+    """ Gets the information about the gym. """
+    return _gym_info.get(gym_id, _default_gym_info)
+
+
+def check_gym(questionable_input):
+    if get_gym_info(questionable_input)['name'] != "unknown":
+        print "Gym found = {}".format(get_gym_info(questionable_input)['name'])
+        return questionable_input, get_gym_info(questionable_input)['name']
+    else:
+        print "Not a valid gym. Please try again.."
```

Comment:
Should follow the standards with the rest of the program of the newline and the `>` since this could happen a lot 😄 
Ex. `print "Not a valid gym. Please try again..\n>",